docs/tab-widgets/configure-agent.asciidoc

```diff
@@ -11,6 +11,7 @@ for more information and a configuration reference.
 For a full list of agent configuration options, see the relevant agent reference:
 
 * {apm-go-ref-v}/configuration.html[Go Agent configuration]
+* {apm-ios-ref-v}/configuration.html[iOS Agent configuration]
```

Comment:
Broken in preview but fixed in https://github.com/elastic/observability-docs/commit/cc8dc2a89374207f6446c2afaa4ad35cb15c6361